pkg/cmd/opts/install.go

```diff
@@ -780,7 +780,35 @@ func (o *CommonOptions) InstallGcloud() error {
 	if runtime.GOOS != "darwin" || o.NoBrew {
 		return errors.New("please install missing gcloud sdk - see https://cloud.google.com/sdk/downloads#interactive")
 	}
-	return o.RunCommand("brew", "cask", "install", "google-cloud-sdk")
```

Comment:
There is a conceptual difference here which is quite important. `brew cask install` will install `gcloud` in a way that it is automatically in the user's path. He is directly able to execute it. This is not the case for the current code installing into _JX_HOME/bin_. The user would have to add the jx bin directory to his path before being able to use it. 

Do we install _gcloud_ in order for the user to use it directly or do we need to install _gcloud_ because `jx` needs to call it. For the latter installing into _JX_HOME/bin_ is no issue, since we can control the PATH when we are calling it. If we want to help the user to install _gcloud_ for general use this might not be the best option. 

Eventually, the use of Homebrew needs to go (see #6661), but maybe a better solution, for now, is to keep using `brew` for macOS and in the other cases use the approach below (basically aligning the code with _InstallHelm_). @ccojocar WDYT?

Another thing I was hoping to solve as part of #6661 is how we version the binary. As a Jenkins X user you might be working with multiple Jenkins X installs in slightly different versions. As you switch between various clusters you would theoretically have to change the versions in _JX_HOME/bin_. For that to happen the binaries would need to be installed by namespace one way or the other. At least one would need to check/validate the binary version before using it.

The whole area is quite tricky and I am not quite sure how to move forward here. The generic install code below is valuable for sure, I am just not sure what the best way is to integrate it.

Reply:
> At least one would need to check/validate the binary version before using it.

I am quite new to the jenkins X codebase here so forgive me for the inexperience but validation of the binary before use for different versions sounds a viable solution since the user would end up changing the versions "theoretically".

Reply:
> I am quite new to the jenkins X codebase here so forgive me for the inexperience 

No problem.

> but validation of the binary before use for different versions sounds a viable solution

Potentially viable. I think, however, we can defer this problem to #6661. It is probably best to "fix" this for all binary installs together. It is really an orthogonal issue. Maybe once we have this PR merged, you are interested to take this further? 
For now let's install the binary into _JX_HOME/bin_ the same way the other functions in this file do it. I was just providing some additional context related to this change.

Reply:
@hferentschik I am definitely interested to take this further, if you have any related issues to take a look at like #6661 I would love to get started.

Reply:
The idea after un-tzipping the downloaded file is to copy just the binary in the JX bin folder and delete the rest.